cpp/include/Ice/FactoryTable.h

```diff
@@ -57,11 +57,11 @@ class ICE_API FactoryTable : private IceUtil::noncopyable
     mutable std::mutex _mutex;
 
     typedef ::std::pair<::Ice::UserExceptionFactory, int> EFPair;
-    typedef ::std::map<::std::string, EFPair> EFTable;
+    typedef ::std::map<::std::string, EFPair, std::less<>> EFTable;
```

Comment:
std::less<> to be able to lookup using a string_view. 

See https://stackoverflow.com/questions/35525777/use-of-string-view-for-map-lookup

Reply:
Might be nice to add a comment for this.